slack_driver/tests/requirements.txt

```diff
@@ -0,0 +1,8 @@
+credstash
```

Comment:
Duplicated requirements.  Recommend adding your test centric depends to setup.py and consolidating.